ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md

```diff
@@ -130,125 +115,75 @@ Install the dependencies for the four packages which you acquired from Git. For
 
 **Example:**
 
-```
-# apt-get install debhelper
-# apt-get install dh-autoreconf
-# etc. etc. etc. 
-```
+- **`# apt-get install debhelper dh-autoreconf etc. etc. etc.`** 
+
 
 The following packages that need to be installed:
-```
-# apt-get install debhelper
-# apt-get install dh-autoreconf
-# apt-get install fonts-droid-fallback
-# apt-get install gnome-common
-# apt-get install gnome-settings-daemon-dev
-```
+
+- **`# apt-get install debhelper dh-autoreconf fonts-droid-fallback gnome-common gnome-settings-daemon-dev`**
+
 
 In order to install libayatana you will need to update the packages type following:
-```
-# apt-get update
-```
+
+- **`# apt-get update`**
+
 
 Now carry on with `apt-get install` followed by the command.
 
-```
-# apt-get install libayatana-ido3-0.4-dev
-# apt-get install libayatana-indicator3-dev
-# apt-get install libcanberra-dev
-# apt-get install libgnome-desktop-3-dev
-# apt-get install libgtk-3-dev
-# apt-get install liblightdm-gobject-1-dev
-# apt-get install valac
-# apt-get install xvfb
-# apt-get install fonts-cantarell
-# apt-get install fonts-cabin
-# apt-get install gnome-settings-daemon
-```
+- **`# apt-get install libayatana-ido3-0.4-dev libayatana-indicator3-dev libcanberra-dev libgnome-desktop-3-dev libgtk-3-dev liblightdm-gobject-1-dev valac xvfb fonts-cantarell fonts-cabin
+gnome-settings-daemon`**
 
 Install libpam-x2go dependencies, use the same command `apt-get install` followed by the dependencies:
 
-```
-# apt-get install libgtest-dev
-# apt-get install libpam0g-dev
-# apt-get install libssh-dev 
-# apt-get install cdbs 
-# apt-get install socat
-# apt-get install pyhoca-cli
-```
+- **`# apt-get install libgtest-dev libpam0g-dev libssh-dev cdbs socat pyhoca-cli`**
  
 Install the lightdm-remote-session-x2go dependencies, use the same command `apt-get install` followed by the dependencies:
 
-```
-# apt-get install dh-apparmor
-```
+- **`#apt-get install dh-apparmor`**
  
 install remote-logon-service dependencies, use the same command `apt-get install` followed by the dependencies:
 
-```
-# apt-get install dbus-test-runner  
-# apt-get install libdbustest1-dev
-# apt-get install libgcrypt11-dev 
-# apt-get install libjson-glib-dev 
-# apt-get install libnm-glib-dev
-# apt-get install libsoup2.4-dev
-```
+- **`# apt-get install dbus-test-runner libdbustest1-dev libgcrypt11-dev libjson-glib-dev libnm-glib-dev
```

Comment:
bogus format, use backslashes and line-wrap the package names (see example above)